### PR TITLE
Expand DBT crawlers to cover tests macros [sc-3617]

### PR DESCRIPTION
**Pull request by alyiwang**
### Why?

Fill in data for dbt model Virtual View

### What?

- Update dbt crawler to parse and fill dbt models
- Update manifest schema to v2
- Update the test manifest json to be latest one generated with dbt 0.20.2 on "trial" project

### Checklist

- [x] I have tested that the changes in this PR work as expected
- [x] I have added/updated tests that exercise the critical code paths in this diff

### CODE REVIEW

**Comment by mars-lan** [comment]
CI broken?

**Comment by mars-lan** [comment]
LGTM at a high level. Let's fix the CI issue and get this in for the demo!

**Comment by mars-lan** [comment]
Please also test against the manifest files for `shopify` & `ride_share` projects as a sanity check.

File: tests/dbt/test_extractor.py [hunk 1]
```diff
@@ -8,10 +8,42 @@
...
+    extractor = DbtExtractor()
+    events = [EventUtil.trim_event(e) for e in await extractor.extract(config)]
+
+    assert events == load_json(expected)
+
+
+@freeze_time("2000-01-01")
+@pytest.mark.asyncio
+async def test_ride_share_project(test_root_dir):
+    manifest = test_root_dir + "/dbt/data/ride_share_manifest.json"
```
Comment (mars-lan) [inline tests/dbt/test_extractor.py:28]: Instead of doing this, why not `/dbt/data/ride_share/manifest.json` etc?

Comment (alyiwang) [reply tests/dbt/test_extractor.py]: good point

**Review by mars-lan — APPROVED** [review]

**Pull request merged**